GVRf/Framework/jni/engine/importer/assimp_importer.cpp

```diff
@@ -68,4 +71,176 @@ std::shared_ptr<Mesh> AssimpImporter::getMesh(int index) {
 
     return std::shared_ptr < Mesh > (mesh);
 }
+
+void AssimpImporter::sceneRecursion(aiNode* node, const aiScene* aScene, std::shared_ptr<Scene> scnPtr, JNIEnv * env, jobject obj, jobject bitmap, aiMatrix4x4 accumulatedTransform)
```

Comment:
Pretty much all of the JNI code that I've read or touched uses C naming conventions, like `scene_pointer`, not Java conventions like `scenePointer`; the only C++ code I've seen use camelCaps is the contrib/assimp code. I can't say that *all* the GVR JNI code uses the _ `naming_convention`, but enough does that this really stands out. I would very much prefer it if such a large change followed the existing conventions. (I know this sort of feedback can seem petty, but inconsistencies do slow people down - "Why is it different _here_ than _there_?" - and massively inconsistent code is very hard to read.

Second, I'm consistently - almost to the point of fetish - opposed to the use of abbreviations. In a Globish-speaking world, the abbreviation that's obvious to one person might not be at all obvious to another person. It's just not worth slowing down and/or confusing your readers when long names like `scene_pointer` make your code all the clearer and auto-complete makes them just as easy to type. Plus, abbreviations can make bugs harder to spot: I once found a bug where a programmer had used `i` and `j` for loop variables, and coded `array[i, j]` where they meant `array[j, i]`.